screens/HomeScreen.tsx

```diff
@@ -0,0 +1,268 @@
+      let pathnumber = (await lastPath) + 1;
+      const todayDate: string = new Date()
+        .toUTCString()
+        .split(" ")
+        .slice(1, 4)
+        .join("");
+
+      await pathInProgressArray.push({
+        number: pathnumber,
+        ang: 138,
+        progress: 15,
+        startData: todayDate,
+        endDate: "",
+      });
+      await AsyncStorage.setItem(
+        "PathInProgress",
+        JSON.stringify(pathInProgressArray)
+      );
+      setOnGogingPath(
+        pathInProgressArray.filter((path: Path) => path.ang != 1438)
+      );
+    } catch (e) {
+      console.log(e);
+    }
+  };
+
+  return (
+    <>
+      <ImageBackground
+        source={require("../images/homeScreenBackground.png")}
+        imageStyle={styles.backgroundImage}
+      >
+        <View style={styles.container}>
+          <View style={styles.headingContainer}>
+            <Text style={styles.header}>It's a Fine day to start a</Text>
+            <Text style={styles.header}> new Sehaj Path!</Text>
+          </View>
+          <View style={styles.startButtonContainer}>
+            <LinearGradient
+              start={{ x: 0, y: 0 }}
+              end={{ x: 1, y: 0 }}
+              colors={["#11336A", "#0D2346"]}
+              style={{
+                ...styles.startButton,
+                width: width > 768 ? 140 : 112,
+                height: width > 768 ? 55 : 48,
+              }}
+            >
+              <TouchableOpacity
+                style={{
+                  flexDirection: "row",
+                  alignItems: "center",
+                  justifyContent: "center",
+                }}
+                onPress={handleNewPath}
+              >
+                <Image
+                  source={require("../images/pathStartButtonIcon.png")}
+                  style={{
+                    ...styles.pathStartButtonIcon,
+                    height: width > 768 ? 30 : 24,
+                    width: width > 768 ? 30 : 24,
+                  }}
+                />
+                <Text
+                  style={{
+                    ...styles.startButtonText,
+                    fontSize: width > 768 ? 20 : 16,
+                  }}
+                >
+                  START
+                </Text>
+              </TouchableOpacity>
+            </LinearGradient>
+          </View>
+
+          {onGoingPath.length != 0 ? (
+            <>
+              <View style={styles.sectionTitleContainer}>
+                <Text
+                  style={{
+                    ...styles.sectionTitle,
+                    fontSize: width > 500 ? 21 : 14,
+                  }}
+                >
+                  Sehaj Path in Progress:
+                </Text>
+              </View>
+              <ScrollView
+                horizontal
+                showsHorizontalScrollIndicator={false}
+                style={styles.horizontalScroll}
+              >
+                {onGoingPath.length != 0
```

Comment:
!= 0 could be interpreted differently in other contexts, but > 0 unambiguously means “there is at least one element in the array.
use
`(onGoingPath.length > 0)`